Website/DesktopModules/Hotcakes/Core/Admin/Parts/PaymentMethods/PaypalExpress/Edit.ascx

```diff
@@ -15,14 +15,9 @@
 <div class="hcColumnLeft" style="width: 50%;">
     <div class="hcForm">
         <div class="hcFormItem">
-            <asp:Label runat="server" resourcekey="APIUsername" CssClass="hcLabel" />
-            <asp:TextBox ID="txtUsername" runat="server" />
-            <asp:RequiredFieldValidator ID="rfvUsername" runat="server" resourcekey="rfvUsername" ControlToValidate="txtUsername" Display="Dynamic" CssClass="hcFormError" />
-        </div>
-        <div class="hcFormItem">
-            <asp:Label runat="server" resourcekey="APIPassword" CssClass="hcLabel" />
-            <asp:TextBox ID="txtPassword" runat="server" />
-            <asp:RequiredFieldValidator ID="rfvPassword" runat="server" resourcekey="rfvPassword" ControlToValidate="txtPassword" Display="Dynamic" CssClass="hcFormError" />
+            <asp:Label runat="server" resourcekey="APIClientId" CssClass="hcLabel" />
+            <asp:TextBox ID="txtClientId" runat="server" />
+            <asp:RequiredFieldValidator ID="rfvClientId" runat="server" resourcekey="rfvClientId" ControlToValidate="txtClientId" Display="Dynamic" CssClass="hcFormError" />
```

Comment:
I don't see a localization key/value pair for the error message.

Reply:
Please let me know if it is well know.